modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionServiceTest.java

```diff
@@ -168,6 +168,26 @@ public void testUpdateCustomObjectDefinition() throws Exception {
 		_testUpdateCustomObjectDefinition(_user, _user);
 	}
 
+	@Test
+	public void testUpdateRootObjectDefinitionId() throws Exception {
+		try {
+			_testUpdateRootObjectDefinitionId(_adminUser, _user);
+
+			Assert.fail();
+		}
+		catch (PrincipalException.MustHavePermission principalException) {
+			String message = principalException.getMessage();
+
+			Assert.assertTrue(
+				message.contains(
+					"User " + _user.getUserId() +
+						" must have UPDATE permission for"));
+		}
```

Comment:
@gabrielwas here we can use assertFailure as well, for example:

```suggestion
		AssertUtils.assertFailure(
			PrincipalException.MustHavePermission.class,
			"User " + _user.getUserId() + " must have UPDATE permission for",
			() -> _testUpdateRootObjectDefinitionId(_adminUser, _user));
```